Uno/CardUtils.swift

```diff
@@ -11,6 +11,12 @@ import Foundation
 class CardUtils {
     private static var cardDeck = [Card?](repeating: nil, count:108)
     
+    // Predefined action cards
+    static let reverseCard = Card(cardColor: CardColor.other, cardValue: SpecialVals.reverse.rawValue)
```

Comment:
See new commit [https://github.com/aacalfa/iOS-Uno/pull/20/commits/d7293fca0db6d8c07aea14ff14f10ad280cb4417](https://github.com/aacalfa/iOS-Uno/pull/20/commits/d7293fca0db6d8c07aea14ff14f10ad280cb4417) for changes. Wild and Wild Draw Four are special cards. These are variables are simply for the ease to refer to such cards.